app/safe/internal/bootstrap/k8s.go

```diff
@@ -51,7 +50,7 @@ func persistKeys(privateKey, publicKey, aesSeed string) error {
 	data["KEY_TXT"] = ([]byte)(keysCombined)
 
 	// Update the Secret in the cluster
-	_, err = k8sApi.CoreV1().Secrets(vsecmSystemNamespace).Update(
+	_, err = k8sApi.CoreV1().Secrets(env.SystemNamespace()).Update(
```

Comment:
better to use a method than caching a local variable.

Again, clean is better than clever.

Using a method means, we’ll always get the uncached value.
And there is negligible (if any) performance impact to that.

Reply:
It's fine to call SystemNamespace() here, rather than caching variable since persistKeys is called only once. But if it's multiple calls, I would rather cache the value and reuse. Because it will never change!!